index.html

```diff
@@ -61,7 +65,33 @@ <h2>Mishal Firdous</h2>
         </div>
       </div>
     </div>
-    <script></script>
+    <script>
+      
+      function displayLocalStorage () {
+        const experienceList = document.getElementById("experienceList");
+        console.log(experienceList);
+       const localData =  JSON.parse(localStorage.getItem("experiencedata"));
+       localData.forEach((item)=>{
+       //console.log(item.companyName);
```

Comment:
we should not have unnecessary comments in our code base

Reply:
resolved!